### ISSUE TITLE
[5pt] Develop file check that runs at end of gms_pipeline

### ISSUE THREAD

**Issue by CarsonPruitt-NOAA** [NOAA-OWP/inundation-mapping#662]
The log file tells me that something happened in `clip_rasters_to_branches.py` or immediately afterwards while trying to get the raster metadata . It concerns me that this did not throw an error.

```
##########################################################################
Processing branch_id: 1854000001 in HUC: 03160106 ...
##########################################################################

##########################################################################
Subsetting vectors to branches 03160106 1854000001
##########################################################################
Thu Aug 18 18:59:20 UTC 2022
Querying NWM streams ...
Querying NWM catchments ...
Querying NWM Dissolved Levelpaths headwaters ...
Cumulative Time = 0 min(s) and 1 sec

##########################################################################
Clipping rasters to branches 03160106 1854000001
##########################################################################
Thu Aug 18 18:59:21 UTC 2022
/usr/local/lib/python3.8/dist-packages/geopandas/_compat.py:84: UserWarning: The Shapely GEOS version (3.10.2-CAPI-1.16.0) is incompatible with the GEOS version PyGEOS was compiled with (3.8.0-CAPI-1.13.1). Conversions between both will be slow.
  warnings.warn(
Loading file
======/data/outputs/fim_4_0_7_0/03160106/branch_polygons.gpkg
Number of df rows = 46
  0%|          | 0/2 [00:00<?, ?it/s]Clipping 'dem_meters_0' to branch polygons ...
 50%|█████     | 1/2 [00:12<00:12, 12.33s/it]Clipping 'flowdir_d8_burned_filled_0' to branch polygons ...
100%|██████████| 2/2 [00:20<00:00, 10.95s/it]
```

**Comment by BradfordBates-NOAA** [comment]
Hmm, I am seeing an REM grid for this branch:

![image](https://user-images.githubusercontent.com/69594899/186470805-886956b9-4f7f-4d37-bc7e-7ecf86c18d7c.png)

**Comment by CarsonPruitt-NOAA** [comment]
This seems to be resolved by Ryan's hotfix (#665), so I'm closing this for now.
